packages/block-editor/src/components/global-styles/test/use-global-styles-output.js

```diff
@@ -706,7 +706,8 @@ describe( 'global styles renderer', () => {
 			const style = { spacing: { blockGap: '12px' } };
 
 			const layoutStyles = getLayoutStyles( {
-				tree: layoutDefinitionsTree,
+				layoutDefinitions:
+					layoutDefinitionsTree.settings.layout.definitions,
```

Comment:
>  I guess if we leave the test ones we won't have to update the tests every time the real ones change 😅

Yeah, for the `getLayoutStyles` tests, I thought I'd leave it with the hardcoded definitions in the test, since it'll still accept a `layoutDefinitions` object being passed in, as it saves us a few copy/pastes when we need to update in the future.

I reckon we could probably tidy up these tests a little more in follow-ups, but since we don't _need_ to do it just yet, I thought I'd try to keep the changeset slightly smaller for now.

Happy to update these if anyone feels strongly about it, though!